lib/modules/dosomething/dosomething_rogue/dosomething_rogue.install

```diff
@@ -118,6 +118,31 @@ function dosomething_rogue_schema() {
     ],
   ];
 
+  $schema['dosomething_rogue_signups'] = [
+    'description' => 'Table for tracking phoenix sid and corresponding rogue signups id and event id',
+    'fields' => [
+      'sid' => [
+        'description' => 'The sid of the signup as it is stored in phoenix.',
+        'type' => 'int',
+        'not null' => TRUE,
+      ],
+      'rogue_signup_id' => [
+        'description' => 'The signup id as it is stored in rogue.',
+        'type' => 'int',
+        'not null' => TRUE,
+      ],
+      'created_at' => [
+        'description' => 'The Unix timestamp of when the reportback was stored',
+        'type' => 'int',
```

Comment:
ah you're totally right! Sorry - I was looking at the Rogue tables that have all those as `TIMESTAMP`. I think it's ok as is :)